engine/src/main/battlecode/common/GameConstants.java

```diff
@@ -99,7 +102,7 @@ public class GameConstants {
     public static final int DISTANCE_SQUARED_FROM_HEADQUARTER = 9;
 
     /** The discount factor on the amount of damage a carrier can do based on their capacity */
-    public static final float CARRIER_DAMAGE_FACTOR = 0.2f;
+    public static final float CARRIER_DAMAGE_FACTOR = 1.25f;
```

Comment:
idk how engine uses this but assumign this is intended?

Reply:
we multiplied everything by 10 so really this is a reduction from 2 -> 1.25